public/locales/es/about.json

```
@@ -23,6 +23,12 @@
         },
         "part3": ". ¡Tu patio podría servir para acoger un sensor de calidad de aire!"
     },
+    "sensorsDown": {
+        "heading": "Sensor Not Reporting AQI",
+        "part1": "Hay muchos pasos involucrados en el cálculo del AQI usando la información reportada por los sensores Purple Air que hospedamos en South Gate. La EPA ha descrito los pasos de este algoritmo y seguimos las pautas de la EPA para garantizar que nuestros sensores sean precisos.",
+        "part2": "Los sensores Purple Air son excelentes para nosotros porque son asequibles y relativamente precisos dado su costo. Dicho esto, no son tan confiables como algunos sensores más costosos, lo que significa que a veces los valores que registran no son suficientes para calcular el AQI . Esto podría ser muy temporal (en cuestión de minutos u horas), o podría durar más. Si un sensor ha estado inactivo durante mucho tiempo, probablemente significa que el sensor ya no está conectado a Internet o que se ha encontrado un problema de hardware más grave. ",
```

This suggestion just deals with aligning it with the English, not any actual content/judgement on its quality as a translation. GitHub is a really hard place for that, and @hillaryrguez may be a better source 
```suggestion
        "part2": "Los sensores PurpleAir son excelentes para nosotros porque son asequibles y relativamente precisos dado su costo. Dicho esto, no son tan confiables como algunos sensores más costosos, lo que significa que a veces los valores que registran no son suficientes para calcular el AQI. Esto podría ser muy temporal (en cuestión de minutos u horas) o podría durar más. Si un sensor ha estado inactivo durante mucho tiempo, probablemente significa que el sensor ya no está conectado a Internet o que se ha encontrado un problema de hardware más grave.",
```